.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI Build
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5 # v2.7.0
+        with:
+          fetch-depth: 0
+          ref: main
```

Comment:
You can get rid of the `with` block so that it runs the CI tests against the latest commit on the PR branch:
```suggestion
```

Reply:
nice catch :facepalm: